layers/src/main/scala/geotrellis/layers/Njoiner.scala

```diff
@@ -0,0 +1,49 @@
+package geotrellis.layers
+
+import scala.concurrent.ExecutionContext
+import java.util.concurrent.Executors
+
+import cats.effect.{IO, Timer}
+import cats.syntax.apply._
+
+
+object Njoiner {
```

Comment:
Paging @pomadchin @moradology

Reply:
@jbouffard These are just some util functions, they should be moved to `util.TaskUtils` object just above and that file should be rename `IOUtils` since `Task` refers to earlier scalaz task... come to think of it `njoin` is no longer the name of the function that its using, it is `parJoin`, since this is internal code it should be `private [geotrellis]` since I feel like we're going to change this soonish.